src/components/plotly-chart/index.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import Plot from 'react-plotly.js';
+import deepmerge from 'deepmerge';
+import { connect } from 'react-redux';
+import { dark_preview } from '../../utils/plot-templates/dark';
+import { light_preview } from '../../utils/plot-templates/light';
+import './plotly-chart.css';
+/**
+ * Display plotly chart
+ * @param {object} chartSize Chart dimensions in pixels
+ * @param {object} targetRect event.target.getBoundingClientRect()
+ * @param {boolean} visible Whether to show the tooltip
+ * @param {string} text Tooltip display label
+ */
+const PlotlyChart = ({ theme, data, layout }) => {
+  const hideToolBar = { displayModeBar: false };
```

Comment:
I think this variable might be better named more general as `config` or `plotConfig` (as I can see more things potentially added later for example).

Reply:
I think Liam meant here is to change the `hideToolBar` variable to a more general variable name such as `plotConfig`, which can be done here as it's only used as a variable to be passed into the plotly component.

Reply:
yes, i have done exactly in the final PR. where it has to displayModeBar for modal but hide it for preview. I have done a slightly different implementation in that but used plotConfig instead of specific name